nemo/collections/tts/torch/data.py

```diff
@@ -462,6 +472,30 @@ def get_log_mel(self, audio):
             log_mel = torch.log(torch.clamp(mel, min=torch.finfo(mel.dtype).tiny))
         return log_mel
 
+    def pitch_shift(self, audio, sr, rel_audio_path_as_text_id):
+        audio_shifted_path = Path(self.sup_data_path) / f"{rel_audio_path_as_text_id}_pitch_shift.pt"
+        if audio_shifted_path.exists() and self.cache_pitch_augment:
+            audio_shifted = torch.load(audio_shifted_path)
+            return audio_shifted
+        else:
+            choice1 = np.random.uniform(-4, -1)
+            choice2 = np.random.uniform(1, 4)
+            shift_val = random.choice([choice1, choice2])
+            audio_shifted = librosa.effects.pitch_shift(audio, sr=sr, n_steps=shift_val)
+            # save audio_shifted
+            audio_shifted = torch.tensor(audio_shifted)
+            if self.cache_pitch_augment:
+                torch.save(audio_shifted, audio_shifted_path)
+            return audio_shifted
+
+    def _pad_wav(self, wav):
+        if self.pad_multiple > 1:
+            if wav.shape[0] % self.pad_multiple != 0:
+                wav = torch.cat(
+                    [wav, torch.zeros(self.pad_multiple - wav.shape[0] % self.pad_multiple, dtype=torch.float)]
+                )
+        return wav
```

Comment:
Can this be changed to a more precise name (e.g. `_pad_wav_to_multiple`)?

Reply:
changed name to _pad_wav_to_multiple